runtime/runtime/src/verifier.rs

```diff
@@ -293,12 +294,18 @@ pub(crate) fn validate_actions(
         });
     }
 
+    let mut found_delegate_action = false;
     let mut iter = actions.iter().peekable();
     while let Some(action) = iter.next() {
         if let Action::DeleteAccount(_) = action {
             if iter.peek().is_some() {
                 return Err(ActionsValidationError::DeleteActionMustBeFinal);
             }
+        } else if let Action::Delegate(_) = action {
```

Comment:
Cross-posting this comment to track: https://github.com/near/NEPs/pull/366#discussion_r1046574186

This behavior seems unclear and inconsistent to me.

Reply:
I answered in [the comment](https://github.com/near/NEPs/pull/366#discussion_r1068132736)